### PR TITLE
Remove singleton behavior for Reflection Plugin Class

### PR DESCRIPTION
**Pull request by yash-ni**
Fix for #356 and #361 

Changes:

1) Created a new singleton class "ProtoDescriptorString" that will save the current proto string descriptor and can be later used to initialize the reflection service.

2) Removed singleton behavior of Reflection plugin class "LVProtoServerReflectionPlugin" so that now every server will have its own reflection service and separate descriptor pools.

### CODE REVIEW

File: src/lv_interop.cc [hunk 1]
```diff
@@ -44,6 +44,37 @@ namespace grpc_labview
 {
     grpc_labview::PointerManager<grpc_labview::gRPCid> gPointerManager;
 
+    //---------------------------------------------------------------------
+    //---------------------------------------------------------------------
+ 
+    // Initialize the static members
+    ProtoDescriptorString* ProtoDescriptorString::m_instance = nullptr;
+    std::string ProtoDescriptorString::descriptor_;
```
Comment (ni-sujain) [inline src/lv_interop.cc:52]: how do we cleanup these things when server is stopped

Comment (yash-ni) [reply src/lv_interop.cc:53]: Added functionality to free memory when all servers stop

File: src/lv_proto_server_reflection_plugin.h [hunk 2]
```diff
@@ -39,6 +39,7 @@ namespace grpc_labview
 {
     class LVProtoServerReflectionPlugin : public ::grpc::ServerBuilderPlugin {
     public:
+        LVProtoServerReflectionPlugin(); // hide constructors TODO
```
Comment (ni-sujain) [inline src/lv_proto_server_reflection_plugin.h:42]: remove comment

**Review by ni-sujain — APPROVED** [review]

File: src/lv_proto_server_reflection_plugin.h (hunk 2 above)
Comment (yash-ni) [reply src/lv_proto_server_reflection_plugin.h]: Done

**Pull request merged**